run.py

```diff
@@ -295,12 +297,18 @@ def _validate_profile_options(profile_options: str):
         exit(-1)
 
     found = False
-    Model = load_model_by_name(args.model)
-    if not Model:
-        # try load model from canary
-        Model = load_canary_model_by_name(args.model)
-        if not Model:
-            print(f"Unable to find model matching {args.model}.")
+
+    try:
+        Model = load_model_by_name(args.model)
+    except ModuleNotFoundError:
+        traceback.print_exc()
+        exit(-1)
+    except:
```

Comment:
Capture `ModelNotFoundError` here.
```suggestion
    except ModelNotFoundError:
```

Reply:
cc @JansonJohnPZ  Can you address my comment here?

Reply:
This line is added:
```
except ModelNotFoundError
```

I found the error is caught in run.py. To print the traceback message, I restore the following line:
```
traceback.print_exc()
```